tox.ini

```diff
@@ -26,12 +26,14 @@ usedevelop = True
 install_command = pip install -U {opts} {packages}
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/test-requirements.txt
+       sqlalchemy_utils>0.33.0,<=0.34.0
```

Comment:
You can remove this file since its not related to FLAVORs

Reply:
This is add by Hunter and I see some error after remove it when running "sudo tox -e py27".